compiler-plugin/src/main/java/module-info.java

```diff
@@ -21,4 +21,8 @@
     requires io.ballerina.tools.api;
     requires io.ballerina.parser;
     requires io.ballerina.stdlib.graphql.commons;
+
+    exports io.ballerina.stdlib.graphql.compiler;
+    exports io.ballerina.stdlib.graphql.compiler.schema.generator;
+    exports io.ballerina.stdlib.graphql.compiler.service;
```

Comment:
Do we need to export these two packages?

Reply:
removed the packages with [962a513](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1238/commits/962a5131b3ebd56f7657e53bea1475b6d802ba4b)